app/views/provider_interface/provider_relationship_permissions/success.html.erb

```diff
@@ -11,7 +11,7 @@
     </h2>
 
     <p class="govuk-body">
-      Select <a href="/organisations">Organisations</a> (at the top of every page) to update your permissions.
+      Select <%= govuk_link_to 'Organisations', '/provider/organisations' %> (at the top of every page) to update your permissions.
```

Comment:
Should we use the path helper here?

Reply:
:+1: This was in the other PR - will update now it's merged.

Reply:
Added and rebased here https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/2231/commits/c6cdf577614afac2cc132af319f11c3392ef63fd#diff-797a45328232ed22024a3002790f4897R14